pkg/kafka/auth/options.go

```diff
@@ -49,6 +49,21 @@ const (
 
 	defaultPlainTextUserName = ""
 	defaultPlainTextPassword = ""
+
+	// Scram configuration options
+	scramPrefix                   = ".scram"
+	suffixScramUserName           = ".username"
+	suffixScramAlgorithm          = ".algorithm"
+	suffixScramPassword           = ".password"
+	suffixScramTLS                = ".usetls"
+	suffixScramInsecureSkipVerify = ".insecureSkipVerify"
```

Comment:
same question: `--kafka.producer.tls.skip-host-verify`

Reply:
Oh wow I completely missed that, my apologies. Noting that down! Thanks!